docs/pages/about.tsx

```diff
@@ -379,6 +379,15 @@ const teamMembers: Array<Profile> = [
     about: 'Passionate about videogames and football',
     github: 'apedroferreira',
   },
+  {
+    src: '/static/branding/about/gerda.png',
```

Comment:
I believe that we have an opportunity to make the page load faster. The avatars are 400px wide, but don't we only need 140px (70*2)?

Reply:
That makes sense!